controls/os_spec.rb

```diff
@@ -236,3 +238,23 @@
     its(:group) { should match(/^root|syslog$/) }
   end
 end
+
+control 'os-12' do
+  impact 1.0
+  title 'Detect cpu-vulnerability-directory'
+  desc 'The cpu-vulnerability-directory /sys/devices/system/cpu/vulnerabilities'
```

Comment:
In the description, we could add a link to the kernel.org docs: 

"Check for known cpu vulnerabilities described here: https://www.kernel.org/doc/html/v5.6/admin-guide/hw-vuln/index.html"

Reply:
Done. Thank you!